README.md

```diff
@@ -50,6 +50,13 @@ npm install
 npm run dev
 ```
 
+- Install python dependencies
+
+```
```

Comment:
Nit: code blocks like these can have syntax highlighting when rendered by GitHub, you just need to suffix the opening ticks with the language's file extension. For shell scripts that would be "sh". It won't make a difference for this specific block, but other ones it will.

Reply:
Noted, I think all my snippets are shell commands atm.